src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ITypeSymbolExtensions.ExpressionSyntaxGeneratorVisitor.cs

```diff
@@ -34,7 +32,7 @@ private static TExpressionSyntax AddInformationTo<TExpressionSyntax>(TExpression
 
             public override ExpressionSyntax VisitNamedType(INamedTypeSymbol symbol)
             {
-                if (!TypeSyntaxGeneratorVisitor.TryCreateNativeIntegerType(symbol, out var typeSyntax))
+                if (TypeSyntaxGeneratorVisitor.TryCreateNativeIntegerType(symbol, out var typeSyntax))
```

Comment:
⚠️ @CyrusNajmabadi I believe this was a true bug which we should investigate for impact (likely a test is needed)